api/app/signals/apps/questionnaires/fieldtypes/__init__.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Gemeente Amsterdam
+import inspect
+import re
+import sys
+
+from signals.apps.questionnaires.fieldtypes.base import FieldType
+from signals.apps.questionnaires.fieldtypes.boolean import Boolean
+from signals.apps.questionnaires.fieldtypes.integer import Integer
+from signals.apps.questionnaires.fieldtypes.text import PlainText
+
+__all__ = [
+    'PlainText',
+    'Integer',
+    'Boolean',
+]
+
+_PATTERN_CAMEL_CASE = re.compile(r'(?<!^)(?=[A-Z])')
+
+
+def _init():
+    """Initialize field type mapping caches."""
+    field_type_choices = {}
+    field_type_classes = {}
+
+    for _, item in inspect.getmembers(sys.modules[__name__]):
+        if inspect.isclass(item) and issubclass(item, FieldType) and item != FieldType:
+            key = _PATTERN_CAMEL_CASE.sub('_', item.__name__).lower()
+            if key in field_type_choices:
+                raise Exception(f'Class "{item.__name__}" is bad!')
```

Comment:
Maybe add to the error that the name is registered double (or at least that was the intention of the original check+error message).

Reply:
Added `Already registered!` to the exception message.